client/components/jetpack/jetpack-product-info/faq-list.tsx

```diff
@@ -0,0 +1,46 @@
+import { FAQ } from '@automattic/calypso-products';
+import { FunctionComponent, useCallback } from 'react';
+import { useDispatch } from 'react-redux';
+import FoldableFAQ from 'calypso/components/foldable-faq';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+
+type Props = { items?: FAQ[] };
+
+const JetpackProductInfoFAQList: FunctionComponent< Props > = ( { items } ) => {
+	const dispatch = useDispatch();
+
+	const onToggle = useCallback(
+		( faqArgs: { id: string; isExpanded: boolean } ) => {
+			const { id, isExpanded } = faqArgs;
+			const tracksArgs = {
+				faq_id: id,
+			};
+
+			dispatch(
+				recordTracksEvent(
+					isExpanded ? 'calypso_plans_faq_open' : 'calypso_plans_faq_closed',
```

Comment:
I think we can keep the event names as `calypso_partner_portal_product_details_faq_open` & `calypso_partner_portal_product_details_faq_closed`. WDYT?

Reply:
Good point, since this component will eventually be used elsewhere besides the Partner portal, I renamed the event to`calypso_product_details_faq_open` and `calypso_product_details_faq_closed` instead. This is closer to a suggestion but is not specific to partner portal. wdyt?

Reply:
This is good 👍